modules/nf-core/plink2/het/meta.yml

```diff
@@ -0,0 +1,57 @@
+name: "plink2_het"
+description: Calculate Inbreeding data with plink2
+keywords:
+  - plink2
+  - inbreeding
+  - heterozygous genotypes
+  - homozygous genotypes
+  - F coefficient
```

Comment:
Better! I think the `F` should be lower case

What about thins such as 'population genomics' ? What sort of discilpine is the module primarily used for